ports/test_ghostfat/main.c

```diff
@@ -369,10 +369,12 @@ int DumpDiskImage(void) {
     return ERR_NONE;
 }
 
-int test_main(void)
+int main(void)
 {
     int r;
 
+    uf2_init();
```

Comment:
Very nice!  So much cleaner! :)

Reply:
thanks, yup, I think we could move it to its own tests folder later on :)